eng/pipelines/extra-platforms/runtime-extra-platforms-ioslike.yml

```diff
@@ -90,9 +90,7 @@ jobs:
         - template: /eng/pipelines/common/templates/runtimes/build-runtime-tests-and-send-to-helix.yml
           parameters:
             creator: dotnet-bot
-            compileOnHelix: true
-            interpreter: true
-            testBuildArgs: /p:ArchiveTests=true /p:DevTeamProvisioning=- /p:RunAOTCompilation=true /p:MonoForceInterpreter=true /p:BuildTestsOnHelix=true
+            testBuildArgs: /p:DevTeamProvisioning=- /p:RunAOTCompilation=true /p:MonoForceInterpreter=true
```

Comment:
You need to specify `/p:BuildTestsOnHelix=true` and I think `compileOnHelix: true` here if you want to build the apps on helix (the test build and test send commands need to agree).